app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
Consider checking if the animal is already in the `Animal.alive` list before appending it. This will prevent duplicate entries if the same instance is initialized multiple times.